resources/views/vendor/backpack/base/inc/sidebar_content.blade.php

```diff
@@ -3,7 +3,7 @@
 <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 
 <li class='nav-item'><a class='nav-link' href="{{ backpack_url('dashboard') }}"><i class="nav-icon fa fa-dashboard"></i> {{ trans('backpack::base.dashboard') }}</a></li>
-@if(backpack_user()->type=='admin')
+@if(backpack_user()->type=='admin' or backpack_user()->email=='lucia@stats4sd.org' or backpack_user()->email=='d.e.mills@stats4sd.org')
```

Comment:
Effective way of ensuring that you and I always have the ability to see the 'admin' links in the sidebar :-)

But this leads to a question - what happens when a non-admin user goes straight to one of the Crud pages? e.g. if they just go straight to `/admin/user` by pasting in the url directly?